builtin/common/information_formspecs.lua

```diff
@@ -110,6 +119,116 @@ local function build_privs_formspec(name)
 end
 
 
+-- Checks whether the mod with the given name is a separate mod or a part of a modpack
+local function is_mod_sep_or_part_of_mp(modname)
+	local modpath = core.get_modpath(modname)
+
+	local ptn = DIR_DELIM .. "[^" .. DIR_DELIM .. "]+"
+	local sep_mod_ptn = ptn .. "$"
+	local mp_ptn = ptn:rep(2) .. "$"
+	local mods_ptn = DIR_DELIM .. "mods"
+	local worldmods_ptn = DIR_DELIM .. "worldmods"
+
+	if modpath:match(mods_ptn .. mp_ptn) or modpath:match(worldmods_ptn .. mp_ptn) then
```

Comment:
If should be more reliable to get the parent path and use `io.open` to check the presence of `modpack.conf` or `modpack.txt`, similar to how this is done in [pkgmgr.lua](https://github.com/Andrey2470T/minetest/blob/master/builtin/mainmenu/pkgmgr.lua#L117), [2](https://github.com/Andrey2470T/minetest/blob/master/builtin/mainmenu/pkgmgr.lua#L144)